packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -711,8 +710,11 @@ export default {
 }
 
 .c-nav-list-btn-text {
+    @include font-size($nav-text-size);
+    color: $nav-text-color;
+    font-weight: $font-weight-regular;
     @include media('>mid') {
-        margin-top: 2px;
+        font-weight: $nav-text-weight;
```

Comment:
Super NIT – can you add an extra line break between normal CSS rules and any nested classes or media queries etc.

It just makes it easier to scan the styles (and it's something we can't enforce in stylelint)